woeip/apps/core/migrations/0001_initial.py

```diff
@@ -1,5 +1,7 @@
-# Generated by Django 2.1.3 on 2018-11-19 07:45
```

Comment:
There shouldn't be any changes to this file.

Reply:
Migrations are not ignored since they are code and can be manually modified.

No changes are needed to this file. You added imports, but they are not used.

Reply:
Ah that's right!